frameworks/MLNet/exec.py

```diff
@@ -99,7 +99,7 @@ def run(dataset: Dataset, config: TaskConfig):
                     dataset=dataset,
                     output_file=config.output_predictions_file,
                     predictions=prediction_df['PredictedLabel'].values,
-                    truth=dataset.test.y,
+                    truth=dataset.test.y_enc.astype('int'),
```

Comment:
this will fail in regression tasks.
```suggestion
                    truth=dataset.test.y_enc,
                    target_is_encoded=True,
```

Reply:
The predicting code for regression is in another block which doesn't use encoded y. So I think regression won't be broken?